integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxMarketplaceImageTests.java

```diff
@@ -0,0 +1,120 @@
+
+import com.sequenceiq.it.cloudbreak.client.ImageCatalogTestClient;
+import com.sequenceiq.it.cloudbreak.client.SdxTestClient;
+import com.sequenceiq.it.cloudbreak.cloud.v4.CloudProvider;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.ClouderaManagerTestDto;
+import com.sequenceiq.it.cloudbreak.dto.ClusterTestDto;
+import com.sequenceiq.it.cloudbreak.dto.ImageSettingsTestDto;
+import com.sequenceiq.it.cloudbreak.dto.InstanceGroupTestDto;
+import com.sequenceiq.it.cloudbreak.dto.imagecatalog.ImageCatalogTestDto;
+import com.sequenceiq.it.cloudbreak.dto.sdx.SdxInternalTestDto;
+import com.sequenceiq.it.cloudbreak.dto.stack.StackTestDto;
+import com.sequenceiq.it.cloudbreak.dto.telemetry.TelemetryTestDto;
+import com.sequenceiq.it.cloudbreak.exception.TestFailException;
+import com.sequenceiq.it.cloudbreak.log.Log;
+import com.sequenceiq.it.cloudbreak.util.spot.UseSpotInstances;
+import com.sequenceiq.sdx.api.model.SdxClusterStatusResponse;
+import com.sequenceiq.sdx.api.model.SdxDatabaseAvailabilityType;
+import com.sequenceiq.sdx.api.model.SdxDatabaseRequest;
+
+public class SdxMarketplaceImageTests extends PreconditionSdxE2ETest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SdxMarketplaceImageTests.class);
+
+    @Inject
+    private SdxTestClient sdxTestClient;
+
+    @Inject
+    private ImageCatalogTestClient imageCatalogTestClient;
+
+    @Test(dataProvider = TEST_CONTEXT)
+    @UseSpotInstances
+    @Description(
+            given = "there is a running Cloudbreak",
+            when = "a valid SDX create request is sent (latest Marketplace Image)",
+            then = "SDX should be available AND deletable"
+    )
+    public void testSDXWithMarketplaceImageCanBeCreatedSuccessfully(TestContext testContext) {
+        String sdxInternal = resourcePropertyProvider().getName();
+        String cluster = resourcePropertyProvider().getName();
+        String clouderaManager = resourcePropertyProvider().getName();
+        String imageSettings = resourcePropertyProvider().getName();
+        String stack = resourcePropertyProvider().getName();
+        String masterInstanceGroup = "master";
+        String idbrokerInstanceGroup = "idbroker";
+        String telemetry = "telemetry";
+        CloudProvider cloudProvider = testContext.getCloudProvider();
+        AtomicReference<String> selectedImageID = new AtomicReference<>();
+
+        SdxDatabaseRequest sdxDatabaseRequest = new SdxDatabaseRequest();
+        sdxDatabaseRequest.setAvailabilityType(SdxDatabaseAvailabilityType.NONE);
+
+        String imgCatalogKey = "mp-img-cat";
+        testContext
+                .given(imgCatalogKey, ImageCatalogTestDto.class)
+                .withName(imgCatalogKey)
+                .withUrl("https://gist.githubusercontent.com/daszabo/6b53c1a413d9b6cb847b9ad786e7bbf5/raw/0edb66302b7f485527b3bc03fdff1d68695d354c/" +
```

Comment:
Could you replace this GIST reference with a versioned one in this project at https://github.com/hortonworks/cloudbreak/tree/master/integration-test/src/main/resources. Please create a new sub folder then move the `v3-marketplace-image-catalog.json` to here, like we have for blueprints.